Formula/secman.rb

```diff
@@ -23,6 +23,6 @@ def install
   test do
     system bin/"secman", "init"
     system bin/"secman", "crypto", "HOMEBREW"
-    assert_match /secman v#{version} (\d{4}-\d{2}-\d{2})/, shell_output("#{bin}/secman --version")
+    assert_match(/secman v#{version} (\d{4}-\d{2}-\d{2})/, shell_output("#{bin}/secman --version"))
```

Comment:
```suggestion
    assert_match(/secman v#{Regexp.escape(version)} \(\d{4}-\d{2}-\d{2}\)/, shell_output("#{bin}/secman --version"))
```
We need to add a few escapes here.

Reply:
I have delete this test, because homebrew doesn't want `help` or `version` as formula test